Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gradle:8.5-jdk21 as build
+FROM gradle:8.6-jdk21 as build
```

Comment:
Any particular reason for updating this? I'd rather do this in a separate PR as we try to keep the Gradle versions in sync between the Docker file and the Gradle wrapper.

Reply:
Actually  I can remove that. I had issues running the docker test runner for smoke tests because of this image and when I checked on dockerhub it looked like 8.5-jdk21 wasn't available anymore but that's actually not the case.